darwin.m

```diff
@@ -0,0 +1,67 @@
+// +build darwin
+
+#import <Foundation/NSArray.h>
```

Comment:
Mainly because I can't fully understand elevate. Simple code without other dependency makes it easier to check for security issues. In current implementation, elevated actions only occurs in the helper tool itself.

I'll be glad to see it be improved later, or the cgo headache be resolved the same way of systray.